src/pynxtools/dataconverter/helpers.py

```diff
@@ -810,13 +810,13 @@ def check_for_valid_atom_types(atoms: Union[str, list]):
 
     if isinstance(atoms, list):
         for elm in atoms:
-            if elm not in chemical_symbols:
+            if elm not in chemical_symbols[1::]:
```

Comment:
```suggestion
            if elm not in chemical_symbols[1:]:
```

I also suggest to remove the exception raising and use logging to report that the element was not found.

Reply:
I agree to that in should be captured by the logger, however that is a trickier refactoring that goes beyond the scope of this specific fixing branch. I support that this point should be brought up as an issue that is about "refactor pynxtools nomad nexus parser for using consistent logging and exception handling" key point is: the here mentioned helper function is isolated from the context of the nexus parser class that has a well-defined own logger. Sure one could make a quick solution and pass the logger to the function but I think this is dirty for two reasons: i) this function discussed here has a well-defined task it extracts a string of atom types from a chemical formula if that creates an issue that function should have an own logger but then we have many loggers, the function is used as far as I know only in the pynxtools nexus parser, i.e. therefore it could also well be a class member function of a utility class that gets its own logger that is then connected to the parser logger function,
ii) also there are repetitive checks in several places whether the element is a valid one, I see a point that given the limited usage of the function that it should not at all be required to check if the chemical formula is build from only correct elements, if one could assume that these checks here are even redundant, still of course other issues can happen during the function call and there indeed that function should get a logger, but as I said that function should become a member function of a utility class UNLESS this function is planned to be used outside the pynxtools project for doing this utility task but that execution can again be in a different context with again own different loggers, long short: this is an own issue unrelated to the issue fixed in this pr

Reply:
You don't need to pass the logger to the function. You can just import `pynxtools.logger` (I'm not sure if this is the exact location) and just use it from there.

We already discussed that we should not raise in pynxtools, because this breaks the parsing of the file completely in nomad, so we should adapt to this here accordingly.

Reply:
Ah no sorry, we just use normal python logging, see here for an example https://github.com/FAIRmat-NFDI/pynxtools/blob/aec8ab9f070295d38411b666417500bcd2d25a49/src/pynxtools/dataconverter/helpers.py#L45

Reply:
implemented